site-cookbooks/gemstash/recipes/default.rb

```diff
@@ -0,0 +1,171 @@
+#
+# Cookbook Name:: gemstash
+# Recipe:: default
+#
+# Copyright 2017, Harald Sitter
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+username = 'gemstash'
+groupname = username.clone
+userhome = "/home/#{username}/"
+systemd_dir = "#{userhome}/.config/systemd/user"
+
+repourl = 'https://github.com/pangea-project/pangea-gemstash'
+clonedir = "#{userhome}/src"
+
+ohai 'reload_passwd' do
+  action :nothing
+  plugin 'etc'
+end
+
+group groupname do
+  action :create
+end
+
+user username do
+  home userhome
+  group groupname
+  manage_home true
+  action :create
+  notifies :reload, 'ohai[reload_passwd]', :immediately
+end
+
+systemd_unit 'gemstash.service' do
+  user username
+  # action %i(reload disable)
+end
+
+systemd_unit 'gemstash.socket' do
+  user username
+  # action %i(enable restart)
```

Comment:
you need to apply an action otherwise the socket isn't going to be enabled after provisioning

Reply:
fixed